packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -1073,6 +1157,26 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
         );
     };
 
+    renderUpdateTooltipText = (): ReactNode => {
+        const { nounPlural } = this.props;
+
+        if (this.useDeprecatedUI) return this.renderUpdateTooltipTextDeprecatd();
+
+        return (
+            <>
+                <p>
+                    By default, import will update existing {nounPlural} based on the file provided. The operation will
+                    fail if there are new {this.capIdsText} being imported.
+                </p>
+                <p>
+                    When the "Allow new {nounPlural}" checkbox is checked, data will be updated for matching{' '}
+                    {this.capIdsText}, and new {nounPlural} will be created for any new {this.capIdsText} provided. Data
+                    will not be changed for any columns not in the imported file.
```

Comment:
I think the intent is that this help text would be updated slightly. There's a discrepancy in the spec, though, with different example text shown, so you should probably check with @labkey-robert for clarification.

Reply:
checked with @labkey-robert and wait to update until he mades a decision.